test/tests/functional/pbs_reliable_job_startup.py

```diff
@@ -1797,12 +1799,11 @@ def test_t2(self):
                                   "6gb", 8, 3,
                                   self.job1_place,
                                   self.job1_sel_esc)
-
         self.momA.log_match("Job;%s;task.+started, hostname" % (jid,),
-                            n=10, max_attempts=60, interval=2, regexp=True)
+                            n=10, max_attempts=60, interval=5, regexp=True)
 
         self.momA.log_match("Job;%s;copy file request received" % (jid,),
-                            n=10, max_attempts=10, interval=2)
+                            n=10, max_attempts=60, interval=5)
```

Comment:
remove max_attempts if it is 60.

Reply:
Maybe just remove max_attempts altogether, and just let the default happen, which I think is now 60. This default might actually be increased to 100 to handle slower systems or systems running pbs_cgroups hook (cpuset system)...